ppanggolin/annotate/annotate.py

```diff
@@ -392,24 +401,30 @@ def annotatePangenome(pangenome, fastaList, tmpdir, cpu, translation_table="11",
 def launch(args):
     filename = mkFilename(args.basename, args.output, args.force)
     pangenome = Pangenome()
-    if args.fasta is not None and args.anno is None:
-        annotatePangenome(pangenome, args.fasta, args.tmpdir, args.cpu,  args.translation_table, args.kingdom, args.norna, args.overlap)
+    if args.fasta_list is not None and args.anno is None:
+        pangenome = Pangenome()
+        annotatePangenome(pangenome, args.fasta_list, args.tmpdir, args.cpu, args.translation_table, args.kingdom, args.norna, args.overlap, fastaList=True)
+    elif args.fasta_files is not None and args.anno is None:
+        pangenome = Pangenome()
+        annotatePangenome(pangenome, args.fasta_files, args.tmpdir, args.cpu, args.translation_table, args.kingdom, args.norna, args.overlap, fastaList=False)
     elif args.anno is not None:
         readAnnotations(pangenome, args.anno, pseudo = args.use_pseudo)
         if pangenome.status["geneSequences"] == "No":
-            if args.fasta:
-                getGeneSequencesFromFastas(pangenome, args.fasta)
-            else:
+            if args.fasta_list is None and args.fasta_files is None:
                 logging.getLogger().warning("You provided gff files without sequences, and you did not provide fasta sequences. Thus it was not possible to get the gene sequences.")
                 logging.getLogger().warning("You will be able to proceed with your analysis ONLY if you provide the clustering results in the next step.")
-
+            elif pangenome.status["geneSequences"] == "No" and args.fasta_list is not None:
+                getGeneSequencesFromFastas(pangenome, args.fasta_list)
+            elif pangenome.status["geneSequences"] == "No" and args.fasta_files is not None:
```

Comment:
This is for the case where the user provided annotation through a file listing annotation files, and genome sequences through the --fasta_files option. Do we really want to allow this ?